GridTariffAPI.lib/Services/TariffQueryService.cs

```diff
@@ -155,6 +155,10 @@ public virtual MeteringPointsAndPriceLevels MeteringPointsToPriceLevel(string fi
             DateTimeOffset paramToDate)
         {
             var tariff = _tariffPriceCache.GetTariff(tariffKey);
+            if (tariff == null)
+            {
+                return new GridTariffCollection();
+            }
```

Comment:
ved spørring med utgangspunkt i tariff skal ikke dette skje da det i den utvidede validering av request verifiseres at tariff eksisterer.
Dette kan imidlertid skje i tilfeller hvor det spørres med utgangspunkt i målepunkt, og målepunktet er koblet til tariff som ikke gjenfinnes i prisstrukturen.
Dette bør kanskje vurderes som en feilsituasjon, da vi ikke har sammenheng mellom målepunkt/tariff-informasjon på den ene siden, og tariffpriser på den andre siden.
Burde vi kanskje kaste exception?

Reply:
Vi tar opp tariff-renaming (mismatch i datagrunnlaget) med Are. Beholder null-sjekk, for vi vurderer det som bedre å få manglende info ved mismatch enn internal server error.